FakePrototypeStuff/JoystickInputToEvents.cs

```diff
@@ -21,6 +23,23 @@ public void Update()
 	public const int buttonCount = 20;
 	public const int joystickCount = 10;
 	private float[,] m_LastValues = new float[joystickCount, axisCount + buttonCount];
+
+	static string[] s_Axes;
+
+	static JoystickInputToEvents()
+	{
+		int first = 1;
+		int last = 10;
+		s_Axes = new string[(last - first) * joystickCount + 1];
+		for (int device = 0; device < joystickCount; device++)
+		{
+			for (int i = 0; i <= last - first; i++)
+			{
+				var index = device * (last - first) + i;
+				s_Axes[index] = "Analog" + (i + first) + "_Joy" + (device + 1);
```

Comment:
string concatenation allocates usually because .net strings are immutable, right ?  the compiler should optimize it all into a single string creation, at least, but i think it still allocates

Reply:
This only happens once in the static constructor during the lifetime of the program

Reply:
that's fine then

Reply:
Technically, we're also just not using this class. After I did this experiment, we ended up just removing the JoystickInputToEvents component and the other InputToEvents that we weren't using. We should probably keep this, though, because there's a user PR that addresses the same issue, and this was useful for me to test using Joystick input to avoid the VR SDKs.

Reply:
Another question regarding this point: have we profiled going with the suggested approaches VS going with a string builder+append approach? It's been many years since I've done a profiled comparison, but there was a difference at the time.